sdk/data/azcosmos/partition_key.go

```diff
@@ -14,6 +14,10 @@ type PartitionKey struct {
 	values []interface{}
 }
 
+var NullParitionKey PartitionKey = PartitionKey{
```

Comment:
```suggestion
var NullPartitionKey PartitionKey = PartitionKey{
```

typo on the variable name

Reply:
Resolved this